src/commands/react-native/xcode.ts

```diff
@@ -4,11 +4,17 @@ import {Cli, Command} from 'clipanion'
 import {existsSync} from 'fs'
 import {UploadCommand} from './upload'
 
+/**
+ * Because jest cannot mock require.resolve, reactNativePath cannot
+ * be unit tested. If you make any change to it, make sure to test
+ * it with a real project.
+ */
 const reactNativePath = (() => {
   try {
     const reactNativeIndexFile = require.resolve('react-native')
 
-    return `${reactNativeIndexFile}/..`
+    // We need to remove the trailing "/index.js" at the end of the path
+    return reactNativeIndexFile.split('/').slice(0, -1).join('/')
```

Comment:
will it work on Windows where delimiter is `\`?

Reply:
This script will only be ran from Xcode, which is not available on windows

Reply:
makes sense. Ideally it is better to use something like `Os.Separator` (pseudo-symbol).

Reply:
I changed it to `path.sep` so that it can be more easily reused on other scripts if needed 👍